lms/envs/common.py

```diff
@@ -2701,3 +2701,10 @@
     'options': {},
 }
 PROCTORING_SETTINGS = {}
+
+#### Custom Courses for EDX (CCX) configuration
+
+# This is an arbitrary hard limit.
+# The reason we introcuced this number is because we do not want the CCX
+# to compete with the MOOC.
+CCX_MAX_STUDENTS_ALLOWED = 200
```

Comment:
@pdpinch ^ I'm still curious/concerned about this. Have you had time to figure it out?